### PR TITLE
Conform to README.lint

### PR DESCRIPTION
**Pull request by jonathanpberger**
Proposing changes per [readme-dot-lint](https://github.com/strangelove-ventures/readme-dot-lint).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,60 +1,100 @@
...
 
+We built `interchaintest` to extract patterns and create a generic test harness: a use-case-agnostic framework for generating repeatable, diagnostic tests for every aspect of IBC.
 
+Read more at the [Announcing `interchaintest` blog post](https://strange.love/blog/announcing-interchaintest).
 
-`interchaintest` orchestrates Go tests that utilize Docker containers for multiple
+🌌🌌 Who benefits from `interchaintest`?
+=============================
+
+`interchaintest` is for developers who expect top-shelf testing tools when working in IBC.
```
Comment (boojamya) [inline README.md:30]: I don't think we should include IBC here. 
While ICtest is great for testing IBC, there are many use cases to use it even when users are not testing IBC

File: README.md [hunk 2]
```diff
@@ -1,60 +1,100 @@
...
+
+
+🌌🌌🌌 What does `interchaintest` do?
+=============================
+
+`interchaintest` is a framework for testing blockchain functionality and interoperability between chains, primarily with the Inter-Blockchain Communication (IBC) protocol.
+
+Want to quickly spin up custom testnets and dev environments to test IBC, chain infrastructures, smart contracts, etc.? `interchaintest` orchestrates Go tests that utilize Docker containers for multiple
 [IBC](https://docs.cosmos.network/master/ibc/overview.html)-compatible blockchains.
```
Comment (boojamya) [inline README.md:39]: This link no longer works.

File: README.md [hunk 3]
```diff
@@ -1,60 +1,100 @@
...
+
+🌌🌌🌌🌌 How do I use it?
+=============================
+
+## As a Module
+
+<!-- Most people choose to import `interchaintest` as a module.
+- Ofter, teams will [integrate interchaintest with a github CI/CD pipeline](./docs/ciTests.md).
+- You can also [utilize our suite of Conformance Tests](./docs/conformance-tests-lib.md) to confirm that the basics of IBC work with your chain. -->
+
```
Comment (boojamya) [inline README.md:51]: Should we remove this?

File: README.md [hunk 4]
```diff
@@ -1,60 +1,100 @@
...
+`interchaintest` is for developers who expect top-shelf testing tools when working in IBC.
+
+
+🌌🌌🌌 What does `interchaintest` do?
+=============================
+
+`interchaintest` is a framework for testing blockchain functionality and interoperability between chains, primarily with the Inter-Blockchain Communication (IBC) protocol.
+
+Want to quickly spin up custom testnets and dev environments to test IBC, chain infrastructures, smart contracts, etc.? `interchaintest` orchestrates Go tests that utilize Docker containers for multiple
```
Comment (boojamya) [inline README.md:38]: We should mention how interchaintest can automatically create clients, connections, channels and then spin up a relayer. 

I realize this gets grouped into "IBC" but I think this is one of its most under looked features.

**Comment by boojamya** [comment]
Maybe in a separate PR, but we should absolutly get a link to [local-interchain](https://github.com/strangelove-ventures/interchaintest/tree/main/local-interchain) on the home readme.

File: README.md (hunk 1 above)
Comment (Reecepbcups) [reply README.md]: Generalized to 'blockchain protocols' since we now support ETH too

File: README.md (hunk 4 above)
Comment (Reecepbcups) [reply README.md]: generalized this to ibc Relayer setup with a link to the cosmos/relayer repo

**Comment by Reecepbcups** [comment]
I will do a local-interchain write up in another PR. Should keep this one to basic linting and general info

**Pull request merged**